update.php

```diff
@@ -508,3 +508,12 @@ function update_1351()
 
 	return Update::SUCCESS;
 }
+
+function pre_update_1354()
+{
+	if(!DBA::e("ALTER TABLE `contact` CHANGE `ffi_keyword_blacklist` `ffi_keyword_blocklist` text null")) {
```

Comment:
Have you checked if this worked as expected? This is the first time that we rename fields. And I guess that this won't be the last if this works flawlessly.

And BTW: Could you add a test if the original field exists?

Reply:
This is a query that was generated by my database manager frontend. I'll add the `IGNORE` keyword in case the column doesn't exist under the former name.

Reply:
The `ignore` support had been dropped by Oracle some versions ago. And not all of our users are using MariaDB, I guess.

Reply:
Also the ignore is just for primary key duplicates, not whether the column exists in the first place, I added a condition.